src/ebay-cta-button/cta-button.tsx

```diff
@@ -31,7 +31,7 @@ const EbayCtaButton: FC<Props> = ({
         <a {...rest} className={className} ref={forwardedRef}>
             <EbayButtonCell type="cta">
                 <span>{children}</span>
-                <EbayIcon name="cta" width={8} height={8} />
+                <EbayIcon name="arrowRight24" width={8} height={8} />
```

Comment:
Would `aarowRight12` be a better choice here and then remove the`width` and `height` attributes.

Reply:
am also wondering if we need to render 24 one, shouldn't it to be expected to be same size, sorry - trying to understand height and width on top of it again.

The references am seeing from skin site is like that

https://opensource.ebay.com/skin/#icon

Reply:
Thanks for paying attention to that part, I need to remove `width: 8, height: 8`, cause they don't work anyway. Yeah we can use arrowRight12, cause it's overridden by CSS: 
<img width="160" alt="image" src="https://user-images.githubusercontent.com/2737310/230355117-62997f5b-c1e4-49a6-921f-43f4439847a0.png">